Dockerfile

```diff
@@ -2,7 +2,7 @@
 FROM debian:latest AS build-env
 
 ENV FLUTTER_CHANNEL="stable"
-ENV FLUTTER_VERSION="3.0.1"
+ENV FLUTTER_VERSION="3.7.5"
```

Comment:
@tk-nguyen how to update this, Can we pick your commit in https://github.com/linagora/tmail-flutter/pull/1457?

Reply:
Maybe merge this change first then I'll update my PR later.